File: src/Server/WebApi/Dtos/BookRecommendationDto.cs
namespace Cbc.WebApi.Dtos;

using Cbc.WebApi.Interfaces;
using Cbc.WebApi.Models.Entities;

public class BookRecommendationDto : IMapFrom<BookRecommendation>
{
    public BookDto Book { get; set; } = null!;

    public MeetingSimpleDto Meeting { get; set; } = null!;
}

File: src/Client/src/app/api/models/book-recommendation-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BookDto } from '../models/book-dto';
import { MeetingSimpleDto } from '../models/meeting-simple-dto';
export interface BookRecommendationDto {
  book: BookDto;
  meeting: MeetingSimpleDto;
}
